src/test/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFsUnitTest.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.hellbender.tools.walkers;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by emeryj on 8/25/17.
+ */
+public class CombineGVCFsUnitTest extends BaseTest{
```

Comment:
After https://github.com/broadinstitute/gatk/pull/3475, this should extend `GATKBaseTest`